### PR TITLE
Track tag :^)

### PR DESCRIPTION
**Pull request by WalshyDev**
# Pull Request

## Description:
Added a supreme tag.

## Related Issue:
N/A

### CODE REVIEW

**Review by nhcarrigan — APPROVED** [review]
omg I'm dying

File: src/data/tags.ts [hunk 1]
```diff
@@ -96,4 +96,10 @@ export const tags: Tag[] = [
     title: "UWU?",
     aliases: [],
   },
+  {
+    name: "track",
+    content: "How to track your Hacktoberfest Swag:\n1. Turn off your computer\n2. Go for a walk\n3. Realise that in the grand scheme of things a free shirt is really not that important\n4. Return home\n5. Turn on your computer\n6. Thank <@541305895544422430> for being so awesome",
+    title: "Track your swag",
+    aliases: []
```
Comment (nhcarrigan) [inline src/data/tags.ts:103]:
```suggestion
    name: "track",
    content:
      "How to track your Hacktoberfest Swag:\n1. Turn off your computer\n2. Go for a walk\n3. Realise that in the grand scheme of things a free shirt is really not that important\n4. Return home\n5. Turn on your computer\n6. Thank <@541305895544422430> for being so awesome",
    title: "Track your swag",
    aliases: [],
```
SMH prettier

File: src/data/tags.ts [hunk 2]
```diff
@@ -96,4 +96,11 @@ export const tags: Tag[] = [
     title: "UWU?",
     aliases: [],
   },
+  {
+    name: "track",
+    content:
+      "How to track your Hacktoberfest Swag:\n1. Turn off your computer\n2. Go for a walk\n3. Realise that in the grand scheme of things a free shirt is really not that important\n4. Return home\n5. Turn on your computer\n6. Thank <@541305895544422430> for being so awesome",
+    title: "Track your swag",
```
Comment (nhcarrigan) [inline src/data/tags.ts:103]:
```suggestion
    title: "How do I track my swag?",
```

**Comment by WalshyDev** [comment]
bro how much testing/linting do you have on here? :laughing:

**Comment by nhcarrigan** [comment]
A *lot*. 🙃

**Review by nhcarrigan — APPROVED** [review]
pog there we go

**Pull request merged**